### PR TITLE
[Refactor] Unused code cleanup and compiling warning fix

### PR DESCRIPTION
**Pull request by christorange**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -9,9 +9,7 @@ const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement
 );
 root.render(
-  <React.StrictMode>
-    <App />
```
Comment (IanSaucy) [inline src/index.tsx]: Please add these lines back, we want to keep strictMode while in dev

File: src/screens/Welcome.tsx [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
-      if (foundUser != undefined) {
-        setUser(foundUser);
+      if (foundUser !== undefined) {
+        // setUser(foundUser);
         console.log("User is logged in");
         console.log(foundUser.email);
         navigate("/home");
       }else{
         console.log("User not logged in");
       }
```
Comment (IanSaucy) [inline src/screens/Welcome.tsx]: We should also verify that their credentials are valid. I would assume Firebase provides a way of doing this automatically so let's check into that. In addition, Firebase SDK might have some sort of hook etc so we don't have to manually check for stored credentials in the cache etc. Please look into this and report back with what you find!

File: src/App.tsx [hunk 3]
```diff
@@ -34,16 +34,7 @@ function App() {
...
           <Route path="/portal" element={<Portal/>} /> 
-          <Route path="/getresources" element={<GetResources/>} />           
-          <Route path="/login" element={<Login />} />
-          <Route path="/signup" element={<Register />} />
-          <Route path="/home" element={<Home />} />
-          <Route path="/" element={<Welcome />} />
-          <Route path="/address-info" element={<AddressInfo />} />
-          <Route path="/address-entry" element={<AddressVerify />} />
-          <Route path="/profile" element={<UserProfileScreen />} />
-          <Route path="/interests" element={<Interests />} />
```
Comment (IanSaucy) [inline src/App.tsx:45]: I see you removed these routes. Are they simply not used? Were they replaced somewhere else?

**Review by IanSaucy — CHANGES_REQUESTED** [review]
A few changes and clarifications before merge.

Thanks!

File: src/screens/Welcome.tsx (hunk 2 above)
Comment (IanSaucy) [reply src/screens/Welcome.tsx]: https://firebase.google.com/docs/reference/js/v8/firebase.auth.Auth

Check out the `onAuthStateChanged` function. I think we can leverage this. Check out the useOnAuthState section in this tutorial: https://designcode.io/react-hooks-handbook-firebase-auth

File: src/App.tsx (hunk 3 above)
Comment (christorange) [reply src/App.tsx:45]: They are duplicated, lines above are same routes

File: src/screens/Welcome.tsx (hunk 2 above)
Comment (IanSaucy) [reply src/screens/Welcome.tsx]: @christorange Jason is working on this I think? So no need to include on this PR but make sure it's on Trello.

**Pull request merged**